### PR TITLE
Connect to Benicio via Slack!

### PR DESCRIPTION
**Pull request by crookedneighbor**
# Problem

Too long have we been without link logging capabilities!
# Solution

Port irc-link-logging module to slack.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,118 @@
+{
```
Comment (crookedneighbor) [inline .eslintrc:1]: Should we have a standard Haxiom eslint?

Comment (lejeunerenard) [inline .eslintrc:1]: :+1:

Comment (NickTomlin) [inline .eslintrc:1]: Sounds bueno to me! I'd also be interested in adopting the js [standard](https://github.com/feross/standard) (no semicolons :scream_cat:) that way we wouldn't have to worry as much about maintaining it.

Comment (crookedneighbor) [inline .eslintrc:1]: I'd be willing to try it. Maybe a PR after this one is merged in.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "author": {
+    "name": "Haxiom"
+  },
+  "contributors": [
+    {
+      "name": "Blade Barringer",
+      "email": "blade@crookedneighbor.com"
+    }
+  ],
+  "license": "GPL-2.0",
```
Comment (NickTomlin) [inline package.json]: Any reason we chose GPL instead of MIT? (more curious than anything)

File: src/haxfred-slack-link-logging.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+        deferred.resolve();
+      });
+    } else {
+      deferred.resolve();
+    }
+  });
+}
+
+module.exports = haxfredSlackLinkLogging;
```
Comment (NickTomlin) [inline src/haxfred-slack-link-logging.js:45]: Is there a reason why we are mixing es module syntax with CJS here?

Comment (crookedneighbor) [inline src/haxfred-slack-link-logging.js:45]: Yes. Babel 6 changed the way it handles `export default` 

http://stackoverflow.com/questions/33505992/babel-6-changes-how-it-exports-default

Comment (crookedneighbor) [inline src/haxfred-slack-link-logging.js:45]: We could use this plugin: https://www.npmjs.com/package/babel-plugin-add-module-exports

**Comment by NickTomlin** [comment]
Did not get to give this a fine-toothed look over but it's :+1: for me

File: src/haxfred-slack-link-logging.js (hunk 3 above)
Comment (NickTomlin) [inline src/haxfred-slack-link-logging.js:45]: Ah yes. That's a pickle. I don't particularly like mixing things but this is fine for now.

**Comment by lejeunerenard** [comment]
:rocket:

File: package.json (hunk 2 above)
Comment (crookedneighbor) [inline package.json]: We used this in previous Haxfred modules. I'm fine with MIT. @lejeunerenard Thoughts?

Comment (lejeunerenard) [inline package.json]: I'm ok with either. There are things I like about both licenses but am not drawn to one or the other.

**Pull request merged**